internal/pkg/client/cache/net_test.go

```diff
@@ -34,3 +34,30 @@ func TestNet(t *testing.T) {
 		})
 	}
 }
+
+func TestNetImageExists(t *testing.T) {
+	tests := []struct {
+		name     string
+		sum      string
+		path     string
+		expected bool
+	}{
+		{"empty data", "", "", true},
```

Comment:
this is a personal preference, and we might need a discussion to standarize on this, but...

- There's a linter I'd like to enable eventually that emits an error if you don't use named fields in struct initializers
- Having named fields makes things easier to read, for example here I don't know (without going back and forth between this and the struct declaration) what those two blanks are.

You can write:

```
{ name: "empty data", sum: "", path: "", expected: true}
```

Reply:
I was following the style of what was already there. I will update the entire file, I have no preference on the topic and having a linter complain about it is a good reason to just do it.